src/extensions/http_api/definitions.js

```diff
@@ -123,7 +123,7 @@ module.exports = ({ baseUrl = '' } = {}) => ({ Given, When, Then }) => {
      */
     Then(/^(?:I )?should receive a collection of ([0-9]+) items? for path (.+)$/, function(itemsNumber, path) {
         const { body } = this.httpApiClient.getResponse()
-        const array = _.get(body, path)
+        const array = Helper.getItemFromPath(body, path)
```

Comment:
Instead of trying to use this matcher for counting direct arrays using `.` perhaps we should add another one or play with the regex to handle `I should receive a collection of X items`, we should try hard to keep the definitions human readable.